tests/acceptance/features/coreApiWebdavPreviews/previews.feature

```diff
@@ -246,19 +249,20 @@ Feature: previews of files downloaded through the webdav API
     And user "Alice" has shared folder "FOLDER" with user "Brian"
     And user "Brian" has accepted share "/FOLDER" offered by user "Alice"
     And user "Alice" has downloaded the preview of "/FOLDER/lorem.txt" with width "32" and height "32"
-    And user "Brian" has downloaded the preview of "Shares/FOLDER/lorem.txt" with width "32" and height "32"
+    And user "Brian" has downloaded the preview of shared resource "Shares/FOLDER/lorem.txt" with width "32" and height "32"
     When user "Alice" uploads file "filesForUpload/lorem.txt" to "/FOLDER/lorem.txt" using the WebDAV API
     Then the HTTP status code should be "204"
     And as user "Alice" the preview of "/FOLDER/lorem.txt" with width "32" and height "32" should have been changed
-    And as user "Brian" the preview of "Shares/FOLDER/lorem.txt" with width "32" and height "32" should have been changed
-    When user "Brian" uploads file with content "new uploaded content" to "Shares/FOLDER/lorem.txt" using the WebDAV API
-    Then the HTTP status code should be "204"
+    And as user "Brian" the preview of shared resource "Shares/FOLDER/lorem.txt" with width "32" and height "32" should have been changed
+    When user "Brian" uploads file with content "new uploaded content" to shared resource "Shares/FOLDER/lorem.txt" using the WebDAV API
+    Then the HTTP status code should be "<http-status-code>"
     And as user "Alice" the preview of "/FOLDER/lorem.txt" with width "32" and height "32" should have been changed
-    And as user "Brian" the preview of "Shares/FOLDER/lorem.txt" with width "32" and height "32" should have been changed
+    And as user "Brian" the preview of shared resource "Shares/FOLDER/lorem.txt" with width "32" and height "32" should have been changed
     Examples:
-      | dav-path-version |
-      | old              |
-      | new              |
+      | dav-path-version | http-status-code |
+      | old              | 200              |
+      | new              | 200              |
+      | spaces           | 204              |
```

Comment:
I wouldn't expect to different codes for different paths
```suggestion
      | old              | 204              |
      | new              | 204              |
      | spaces           | 204              |
```

Reply:
The test failed for old and new dav versions giving the status code `200`. So updated the status code.

Reply:
But we should not. Upload to existing file should always give `204`. I found some possible issues in webdav.php. Maybe that was causing this.